CredentialProvider/CredentialAssets.xcassets/clear-icon.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
Duped asset?

Reply:
I think @rilury probably has a more final answer but I _assume_ this is because an app extension can't reference assets from the application it is embedded in. So copying those assets two over is probably the pragmatic thing to do?

Reply:
Yes indeed, since we are developing using an app extension the resources should be added here. I just tested with an image from the app and it won't appear on the device, so I think this is necessary.